sway/ipc-server.c

```diff
@@ -663,7 +663,7 @@ void ipc_client_handle_command(struct ipc_client *client) {
 		// TODO: Check if they're permitted to use these events
 		struct json_object *request = json_tokener_parse(buf);
 		if (request == NULL) {
-			client_valid = ipc_send_reply(client, "{\"success\": false}", 18);
+			client_valid = ipc_send_reply(client, "[{\"success\": false}]", 20);
```

Comment:
Can we use `strlen` instead of hardcoding the string length?

Reply:
Changed all replies in the case to use strlen